src/__stories__/examples/show/schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../../../../ui-schema.json",
+  "type": "form",
+  "title": "Security",
+  "description": "A shared security scheme definition",
+  "fields": {
+    "securityDefinitions.?": {
```

Comment:
Unrelated to the PR, are these properties' names meant to be used as regular expressions, i.e. `new RegExp(expr)`?

Reply:
If so, `.` is inappropriately used here - dot will match any character.
```js
new RegExp('.').test('a'); // true
new RegExp('.').test('.'); // true
```

To fix the issue, you should prepend the dot with slash, i.e. `\\.`
```js
new RegExp('\\.').test('a'); // false
new RegExp('\\.').test('.'); // true
```

Reply:
Nope. They're not regular expressions. They are like a glob-style, wildcard syntax.

Reply:
also, https://xkcd.com/208/

Reply:
also, https://stackoverflow.com/a/1732454 :laughing: